plugins.json

```diff
@@ -2705,6 +2705,66 @@
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl/aeJgACgkQ0bVLR6XO/sSYOQ//Yxs2c1qEFUsG14SSHJA2S/eAt0YW7AcT+hpjmeUQw20nOPjIO89RTTEkhjwIzWWCw1PvideeJICUcLCG/rZsYO5fqmBHWcesrCmx9e3pSa1jMiKjKwiUhbLNfICKwVPbBBjtrTgKCHNrB8ha1x5tHTgv38EzhjvUSAiATLG/8pgy9q0uBISarB6CvkDEaZ0NkXzlXXC/gqWz2yoUGbaLd1jpcsbjCmZPBD+VOhcU6pefHMaQWSS69MjUXMzrm8IT4dl6XcNF7eK/tW1tueOu4br4F1BYA9iea/nESQlYlO13pNlvMLXm8vz+6YkgS3a4kXJBqWcf6WNU/9NseNCUtJTbPoG30Ix0QhsKB+azi7SP7namm0YXmysOmGFiyt5eCYapFyn7EJJXuTYxgTI+i8kjyzVVaT0EAQv/ZzHXwr2cMJMS/Xeg+gz5ChouKwe+OLDb95B+qDOsaN5bVrrqHiRufmjMbzmHZlGmS6j1qm2pgz3lo2t1whpink1ucRuDtpZpXYHwrisyoh6IHUjtcNjFHbckei+iGv9Ugh5BKtxh5ngbMSie4nH9XZx8rlWi0dsOSsS+uMO8LnPtZopTxSoiLAS8dD2W5bCSS/gN/XkFVOx7l/tHh10zkijpYhlagXi5w/Y3IHaY9ZtSWnsS/mg6x/ZuXsl84rUSEup4ulo="
+      },
+      "darwin-amd64": {
+        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-incident-management-v1.2.0-osx-amd64.tar.gz",
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl/aeJUACgkQ0bVLR6XO/sSiTA//W15OvRzb44NPqxfIQWBOn2qpU7i9C1UaEYjDXl8PtfuFffrInwBw8c+O760+EHwt4skp3qogAOYGqj3epfFnbHB8hHAuFAZdx4i0vVdx8atPe6A2Q5nL5ISmeWfh2JUMfZe72UlgR7hFWuS/fd6xqKAwOiBsqslbgzEuxDNZqkIOZyb8fS9ZcMnc+uHsUwnIn/if3Wst1mRQ0WBIX3XQ6ixbF/b8Hx6oS6osQhhMDyW08a5zCOYsrPnKNSwpsUYPzN2oqESRoAUZCvSvCo+hfUIYj3wB6fVcfCc1xjh1N6g1qqd2VHqTvevoGnXlcp2S9odHMknOtbkd6ToSvkqQNRKz7qQgbrJf2sC7+A63QxZqeYbewOKr/WNZapKTCDJKfc+lpHlth5DKwPfGRaMyZg8WyvqgHJt75mu721sBZonYU+iN6WTz3n0sLVgFcCHEDjOE168/vTS2Zhd9RLQ9CBsjB8TW2VtmM8bfI+dCIopyUB90dU2Awu7C9fKmgDKqe2mCGQtd2InRTezCaEOrQzKdwDFpjBSHEFQQlwWp2xltXblA4syfFmeJcyFffXRJkx1LIJc69C5oWlXj3p1o8/cVxU/ji3+nbSLrd2BNdENlFaDNiwgzXOxTPjZNTHmoExbQ2K5YAgWD1OKT51a+dxvxImPmtCblWQmHTPSn69c="
+      },
+      "windows-amd64": {
+        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-incident-management-v1.2.0-windows-amd64.tar.gz",
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAl/aeJYACgkQ0bVLR6XO/sStXQ/9HhBT8BQ19+H2BNhNrmlz7u5F64dflgIpY3180oUshkPXSS2QR43MbxW8dleUdTZgcBMq7uqVfuhAY56MKTJ8guzBDJlT6bo6PRUaIg53AfBd3cQKBl9ViW0QxX7MWP/T9oLATTTN64mUdRZrnH37oIHzymD/tDEqMzvokSZjWpCwflbyHCx2xXShzM1AnKAmlOQ04mVONFGsR0kvNxa9n1tjvxMk1YwrNHKQ7ybrTsjo3fYY9qg1MqFgX3HZRCiEEdT2Cs601tQgcmO2lqQ5TzQgn5/aKeVvb9733Shyd8A/l6AfKVniT5x1xhOUaQiLC6INlbwvoQJkD798UeUSEA0nC6C0oJ2pz25WGde6bYy1KBy6qNUEH/QzEPLmXbneP5VXfU8JA8NvJMdcm4L82+BJcGtbO13udeCkdC3aKkrM9cThLFfLQU81CciZ+Uoq8LIEa7ur0x3OgQ/2l1sPfbSMZ0tETXIg0IA+4puRqBZceb7ED4Ln+H/0q6bOn1EtH1KbzUtOJ2MZTVslWPOKSYbcI/UGGR8ltk8caI4CP4Nt07qgH6bu5FcLTNJhvipT9FzCBrH0X4Fx4/PMSSby3qd17gumF4Tcs9Yd/HKUexKBSVBnULZyTUQCQMdDfR2lQfziX6c2n8iWq4r8zxO/QJk5jzh6tEiPgr0fWIeZ3SM="
+      }
+    },
+    "updated_at": "2020-12-17T01:22:00.432331Z"
+  },
   {
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-incident-response/",
```

Comment:
I noticed that there is still an incident-response here. Is this intended?

Reply:
Yeah, I only just updated the url-related metadata for `plugin.json` in v1.2.0, so the old metadata for the older versions remains incorrect. Given the permanent redirect, I think we can leave this as-is, and it will gradually become less relevant as new versions are released.